templates/Dockerfile-deb.tmpl

```diff
@@ -90,6 +106,18 @@ RUN curl -s https://packagecloud.io/install/repositories/citusdata/community/scr
 COPY make_pg_buildext_parallel.patch /
 RUN patch `which pg_buildext` < /make_pg_buildext_parallel.patch
 
+
+# install cmake from source
```

Comment:
Why do we need to install this from source? Can you add a comment on why it is not sufficient to just install a package?

In case CMake releases a patch version, we will not get that now.

Reply:
For vpkg to work minimum citus 3.22 is required